contracts/script/output/devnet/alignedlayer_deployment_output.json

```diff
@@ -26,4 +26,4 @@
     "alignedLayerUpgrader": "0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266",
     "pauserRegistry": "0xe7f1725E7734CE288F8367e1Bb143E90bb3F0512"
   }
-}
+}
```

Comment:
Add newline

Reply:
Every time the file is generated, there is missing the newline

Reply:
Maybe we should change it to add the newline